CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+2.0 (unreleased)
+================
+
+Other Changes and Additions
+---------------------------
+
+- Added release procedure to the documentation.  [#268]
```

Comment:
The sentence above stating that this is about adding a new format, now no longer makes sense!

Reply:
I don't have a strong opinion about this, but Astropy has its release procedure under ["Developer Documentation"](http://docs.astropy.org/en/stable/#developer-documentation), which makes sense to me (though they also have more stuff in general, so maybe it looks less jarring).

Reply:
Now addressed.